command/delegate/delegate.go

```diff
@@ -303,12 +309,12 @@ func (c *delegateCommand) handleSetup() http.HandlerFunc {
 			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
-		JSON(w, api.SetupResponse{IPAddress: instance.IP, InstanceID: instance.ID}, http.StatusOK)
+		JSON(w, setupResponse{IPAddress: instance.IP, InstanceID: instance.ID}, http.StatusOK)
```

Comment:
```suggestion
		JSON(w, struct {
			InstanceID string `json:"instance_id,omitempty"`
			IPAddress  string `json:"ip_address,omitempty"`
		}{
			IPAddress: instance.IP,
			InstanceID: instance.ID
		}, http.StatusOK)
```

Reply:
Because the struct does not have to exist outside of this function, this one place.